### DISCUSSION TITLE
Why i am no longer able to update the filter lists manually?

### DISCUSSION THREAD

**Discussion by Buffmutti** [uBlockOrigin/uAssets#22275] [Q&A] [answered]
Five days ago i reported a detection problem on Civitai.com and some helpful guys figured out how to get rid of the ads on that site. So i thought they might add this to the filter lists and since i still see ads on that site i thought that i need to update the filter lists.
But surprise, surprise, the button to clear all filter lists is gone and without that you can't manually update these lists.
Why was this button removed?

**Comment by gorhill** [comment, marked as answer]
https://twitter.com/gorhill/status/1747758689035968701

Forcing a download of filter lists accomplish nothing beside burdening remote servers.

The issue re `civitai.com` is [here](https://github.com/uBlockOrigin/uAssets/issues/22203), and I don't see that you provided feedback to people who worked to fix the issue.

If you think it still need to be fixed, provide feedback along with troubleshooting info for the site. If you had provided feedback and said you wanted the fix immediately, would would have been provided a link to get the fix without having to download MBs of lists. On my side, the troubleshooting information shows that the filter is being applied.

The fix was added to filter lists days ago, by now you have it for sure, so if it still doesn't work, something else on your side is interfering with uBO.

**Comment by Buffmutti** [comment]
My mistake. I mistook the blue banners asking me to become a VIP member for the adblocker detection (they look very similar). So yes, the fix works.
